jberet-core/src/main/java/org/jberet/job/model/Flow.java

```diff
@@ -62,7 +70,7 @@ public String getAttributeNext() {
      */
     @Override
     public Listeners getListeners() {
-        throw new IllegalStateException();
+        return new Listeners();
```

Comment:
In here and a few other places where previously `IllegalStateException` was thrown, if we change to `return null`, will it work with Quarkus? If not, can we return a special `Listeners`, or `Properties` static instances so that some error conditions will not be hidden with a normal listeners or properties instance?

For example, in `Properties` class:

```java
    public static Properties emptyProperties() {
        final Properties empty = new Properties();
        //empty.nameValues = Collections.emptyMap();
        empty.nameValues = null;
        return empty;
    }
```

Reply:
Yes, I believe we can return null. Let me give it a try.

Reply:
@radcortez have you got a chance to check it?

Reply:
Sorry, I went on PTO. I'll have a look.

Reply:
Seems to work ok with `null`. Updated the PR.